style.css

```diff
@@ -910,3 +917,1217 @@ footer #contact-form .contact-details p {
+    line-height: 24px;
+    color: #6b778c;
+    margin-left: 1.5rem;
+  }
+
+  .work-card-4 .work-detials .p-desktop {
+    width: 90%;
+    height: 50px;
+    display: flex;
+    flex-wrap: wrap;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 24px;
+    color: #344563;
+    flex: none;
+    order: 1;
+    flex-grow: 0;
+    padding-left: 5px;
+    margin-left: 10px;
+  }
+
+  .work-card-4 .work-detials .tags-desktop {
+    display: block;
+  }
+
+  .work-card-4 .work-detials .tags-desktop .hash {
+    display: flex;
+    list-style: none;
+    flex-direction: row;
+    margin-left: 10px;
+    align-items: center;
+    font-family: 'Poppins', sans-serif;
+    gap: 1rem;
+    margin-top: 2rem;
+  }
+
+  .work-card-4 .work-detials .tags-desktop .hash .HTML {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 4px 10px;
+    width: 53px;
+    height: 24px;
+    background: #ebebff;
+    color: #6070ff;
+    font-weight: 500;
+    font-size: 12px;
+    letter-spacing: 0.03em;
+    border-radius: 8px;
+    order: 0;
+    flex-grow: 0;
+  }
+
+  .work-card-4 .work-detials .tags-desktop .hash .Ruby-on-rails {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 4px 14px;
+    width: 110px;
+    height: 24px;
+    background: #ebebff;
+    color: #6070ff;
+    font-weight: 500;
+    font-size: 12px;
+    letter-spacing: 0.03em;
+    border-radius: 8px;
+    order: 0;
+    flex-grow: 0;
+  }
+
+  .work-card-4 .work-detials .tags-desktop .hash .CSS {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 4px 10px;
+    width: 45px;
+    height: 24px;
+    background: #ebebff;
+    color: #6070ff;
+    font-weight: 500;
+    font-size: 12px;
+    letter-spacing: 0.03em;
+    border-radius: 8px;
+    order: 1;
+    flex-grow: 0;
+  }
+
+  .work-card-4 .work-detials .tags-desktop .hash .JavaScript {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 4px 14px;
+    width: 94px;
+    height: 24px;
+    background: #ebebff;
+    color: #6070ff;
+    font-weight: 500;
+    font-size: 12px;
+    letter-spacing: 0.03em;
+    border-radius: 8px;
+    order: 2;
+    flex-grow: 0;
+  }
+
+  #About {
+    display: flex;
+    margin-top: 6rem;
+    background-image: url("Images/self-desktop1.png");
+    background-size: cover;
+    width: 100%;
+    height: auto;
+    padding-bottom: 10rem;
+  }
+
+  #About .Aboutcontent {
+    padding-top: 8rem;
+    margin-left: 5rem;
+  }
+
+  #About .Aboutcontent h3 {
+    width: 100%;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 2.5rem;
+    line-height: 3.25rem;
+    display: flex;
+    align-items: center;
+    color: #172b4d;
+  }
+
+  #About .Aboutcontent p {
+    width: 447px;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
```

Comment:
- [Optional] I can see that your CSS file is large(has a lot of lines of code). It is best practice to use [DRY principle](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) to reduce your file size. I can see that you repeated most of your CSS properties hence increasing the lines of code. You can create a class for some properties that occur repeatedly and add that class to your HTML elements instead of repeating the properties over and over. For example, I can see that your repeated these properties, so you can create a class instead like;
```css
.poppins-normal-font {
  font-family: 'Poppins', sans-serif;
  font-style: normal;
}
```
And in your HTML file add the class to your p element like
`<p class="poppins-normal-font"></p>`